runtime/tos/tos.go

```diff
@@ -45,15 +45,23 @@ func VerifyTosAcceptedOrPrompt(ctx *cli.Context) error {
 		return nil
 	}
 
+	if err := PromptTos(); err != nil {
```

Comment:
See other comment re prompts - I would advocate for a workflow where we exit with an error when the flag isn't set, avoiding the need for interactive prompts.

Reply:
will revert and create a workflow that doesn't require prompts